plugin/plugin.go

```diff
@@ -876,8 +879,8 @@ func (b *ORMBuilder) parseBasicFields(msg *protogen.Message, g *protogen.Generat
 				fieldType = "*" + generateImport("Time", stdTimeImport, g)
 			} else if rawType == protoTypeJSON {
 				if b.dbEngine == ENGINE_POSTGRES {
-					typePackage = gormpqImport
-					fieldType = "*" + generateImport("Jsonb", gormpqImport, g)
+					typePackage = gtypesImport
```

Comment:
Can we leverage the new JSON datatype from upstream, or is the custom one better (maybe simpler)? https://github.com/go-gorm/datatypes

Reply:
I'm generally concerned over the fact, that we have a pointer to the struct. Thus I can assume, that some application code might rely on checking for `nil` field value, while new `JSON` datatype will hold `"null"` in case scanned from DB and we can't easily distinguish empty slice value from `nil` slice.